ocs_ci/deployment/deployment.py

```diff
@@ -678,81 +675,83 @@ def uninstall_ocs(self):
+            )
 
         elif self.platform == constants.VSPHERE_PLATFORM:
-            image_registry_obj.patch(resource_name='configs.imageregistry.operator.openshift.io',
-                                     params=" '{\"spec\":{\"storage\":{\"emptyDir\":{}}}}' ")
+            image_registry_obj.patch(
+                resource_name='configs.imageregistry.operator.openshift.io',
+                params=" '{\"spec\":{\"storage\":{\"emptyDir\":{}}}}' "
+            )
 
         # Removing the cluster logging operator from OpenShift Container Storage
-        # Remove the ClusterLogging instance in the namespace.
         ocp_obj.exec_oc_cmd("delete clusterlogging instance -n openshift-logging --wait=true --timeout=5m")
 
-        # STEP3: Follow these instructions to ensure that the PVCs listed in the previous step are deleted
-        # STEP 3.1: Determine the pod that is consuming the PVC.
-        pod_list = get_all_pods()
-        for pod in pod_list:
-            for pvc in pvcs_to_delete:
-                if get_pvc_name(pod) == pvc.get('metadata').get('name'):
-                    pod.delete()
+        # Determine the pod that is consuming the PVC.
+        for pvc in pvcs_to_delete:
+            delete_pods(pvc.get_attached_pods())
 
-        # STEP 3.2: Delete the PVCs.
+        # Delete the PVCs.
         for pvc in pvc_list:
             pvc.delete()
             ocp_obj.wait_for_delete(pvc.name)
 
-        # STEP 4: Delete the StorageCluster object.
-        storage_cluster = ocp.OCP(kind=constants.STORAGECLUSTER, resource_name="ocs-storagecluster")
+        # Delete the StorageCluster object.
+        storage_cluster = ocp.OCP(kind=constants.STORAGECLUSTER, resource_name=constants.DEFAULT_CLUSTERNAME)
         storage_cluster.delete()
 
-        # STEP 5: Delete the namespaces and wait till the deletion is complete.
+        # Delete the namespaces
         ocp_obj.delete_project('openshift-storge')
         ocp_obj.wait_for_delete("openshift-storage")
 
-        # STEP 6: List the storage nodes.
-        nodes_list = get_labeled_nodes('cluster.ocs.openshift.io/openshift-storage')
-
-        # STEP 7: For each node, perform the following:
+        # Debugging nodes
+        nodes_list = get_labeled_nodes(constants.OPERATOR_NODE_LABEL)
         for node in nodes_list:
             ocp_obj.exec_oc_debug_cmd(node=node, cmd_list=["rm -rf /var/lib/rook"])
 
-        # STEP 8: Delete the storage classes with an openshift-storage provisioner listed in step 1.
+        # Delete the storage classes with an openshift-storage provisioner list
         for storage_class in sc_list:
             storage_class.delete()
 
-        # STEP 9: Remove the taint from the storage nodes.NOT SUPPORTED YET
+        # Remove the taint from the storage nodes.NOT SUPPORTED YET
         """ ocp_obj.exec_oc_cmd("adm taint nodes --all node.ocs.openshift.io/storage-") """
 
-        # STEP 10: Unlabel the storage nodes.
+        # Unlabel the storage nodes.
```

Comment:
I might be wrong but the next 2 commands are adding a label for openshift-storage and rook?

Reply:
nope, I thought the same at first
when you add - at the end of a label, it does the opposite and removes it

Reply:
oh, missed the '-'